features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -139,6 +142,8 @@
 
     private final EnrichedFlowForwarder enrichedFlowForwarder;
 
+    private final FlowThresholding thresholding;
```

Comment:
Flow thresholding should be an independent feature that is not part of ElasticSearch related functionality. I can imagine having flow threshold without ES persistence.

Reply:
It should. But it depends on the enrichment process, which is only available inside the `elastic` module. Refactoring this would mean to refactor the whole flow processing pipeline and introduce a bunch of shared modules and transport data types. Therefore I will not do this right now. But I will consider this for later.

Reply:
Related issue: https://issues.opennms.org/browse/NMS-13770